index.html

```diff
@@ -0,0 +1,75 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="style.css" />
+    <link
+      rel="stylesheet"
+      href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.8.2/font/bootstrap-icons.css"
+    />
+    <link
+      rel="stylesheet"
+      href="https://fonts.googleapis.com/css?family=Roboto"
+    />
+  </head>
+  <body>
+    <header>
+     <nav>
+       <i class="bi bi-list bi-10x"></i>
+    </nav>
+    <a href=""><h3 id="logo">TM</h4></a>
+   </header>
+    <!--About me section-->
+    <section id="about">
+      <img id="one" alt="white circle" src="images/Ellipse1.svg">
+      <img id="two" alt="white circle" src="images/Ellipse2.svg">
```

Comment:
- [ ] Kindly be noticed that the section background image (decorations are always background images, not `img` tags).